plugins/processors/enum/README.md

```diff
@@ -37,5 +37,5 @@ source field is overwritten.
 
 ```diff
 - xyzzy status="green" 1502489900000000000
-+ xyzzy status="green",status_code=1i 1502489900000000000
++ xyzzy status="green",status_code=1 1502489900000000000
```

Comment:
`1i` is actually correct as it is intended to be stored as an integer, `1` with no `i` means it will be stored as a floating point value.

Reply:
Thanks for the spot.  I almost thought it was a typo and not as a data type.